HeartbeatLogging/Tests/Unit/HeartbeatStorageTests.swift

```diff
@@ -278,23 +278,25 @@ class HeartbeatStorageTests: XCTestCase {
 }
 
 private class StorageFake: Storage {
-  var data: Data?
+  var fakeFile: Data?
   var onRead: (() throws -> Data)?
   var onWrite: ((Data?) throws -> Void)?
 
   func read() throws -> Data {
     if let onRead = onRead {
       return try onRead()
+    } else if let fakeFile = fakeFile {
```

Comment:
nit: the lhs could still be data here

Reply:
Done in [5a48301](https://github.com/firebase/firebase-ios-sdk/pull/9361/commits/5a48301019ef3e64f992a2fea16ceb4a56682233)